generators/config/templates/default.yml

```diff
@@ -56,6 +56,29 @@
  tb_idle_timeout: '180'
  tb_record_screenshot: false
 
+ # -BrowserStack
```

Comment:
Why you have not added new driver here:
 # Specify one of the following drivers: selenium, selenium_dev, webkit, poltergeist, phantomjs, sauce, testingbot